resource_customizations/kubevirt.io/VirtualMachineInstance/health.lua

```diff
@@ -0,0 +1,21 @@
+hs = { status="Progressing", message="No status available"}
+if obj.status ~= nil then
+  if obj.status.phase ~= nil then
+    hs.message = obj.status.phase
+    if hs.message == "Running" then
+      hs.status = "Healthy"
+    elseif hs.message == "Failed" then
+      hs.status = "Degraded"
+    elseif hs.message == "Succeeded" then
+      hs.status = "Suspended"
+    end
+  elseif obj.status.conditions ~= nil then
+    for i, condition in ipairs(obj.status.conditions) do
+      if condition.type == "Ready" and condition.status == "True" then
```

Comment:
This should also set to Degraded if !Ready, and would also be nice to set to Suspended when the Paused condition is set to True.